src/platform/Entropy.cpp

```diff
@@ -22,34 +22,18 @@
  *          on the Linux platforms.
  */
 
-#include <platform/internal/CHIPDeviceLayerInternal.h>
-#include <support/crypto/CHIPRNG.h>
-
-using namespace ::chip;
+#include <crypto/CHIPCryptoPAL.h>
 
 namespace chip {
 namespace DeviceLayer {
 namespace Internal {
 
 CHIP_ERROR InitEntropy()
 {
-    CHIP_ERROR err;
     unsigned int seed;
-
-    // Initialize the source used by CHIP to get secure random data.
-    err = Platform::Security::InitSecureRandomDataSource(getentropy, 64, NULL, 0);
-    SuccessOrExit(err);
-
-    // Seed the standard rand() pseudo-random generator with data from the secure random source.
-    err = Platform::Security::GetSecureRandomData((uint8_t *) &seed, sizeof(seed));
-    SuccessOrExit(err);
+    ReturnErrorOnFailure(chip::Crypto::DRBG_get_bytes((uint8_t *) &seed, sizeof(seed)));
```

Comment:
Previous code had a InitSecureRandomDataSource call which had a platform-dependent argument. Where is that gone?

Generally I expect the change description to explain why this PR removed files and methods and how the new code functions over the old.

Reply:
`InitSecureRandomDataSource` has already been removed long ago. Now we got a platform independent `DRBG_get_bytes`